services/gcp/cloudkms/cryptoKeyVersions.yml

```diff
@@ -0,0 +1,77 @@
+name: Cloud KMS Crypto Key Versions
+description: >-
+  A key version contains key material used for encryption or signing.
+scope: CRITICAL
+notes: >-
+  Cloud KMS is an extremely sensitive service. Keys can be used for encryption/decryption of 
+  sensitive data or for the creation or verification of digital signatures.
+privileges:
+  create:
+    risks: [impact:spend]
+  destroy:
+    risks: [destruction:crypto, impact:encryption]
+    notes: >-
+      Destroyed key versions cannot be recovered.
+      Any data encrypted with the key version will no longer be able to be decrypted.
+  get:
+    risks: [discovery:infra]
+  list:
+    risks: [discovery:infra]
+  manageRawAesCbcKeys:
+    risks: []
+    notes: >-
+      This permission is required to manage AES CBC keys. 
+      It has no risks because it provides no access on its own: the user
+      still needs permissions on the keys.
```

Comment:
Where did you find documentation about this? I wasn't able to confirm.

Reply:
The docs are here https://cloud.google.com/kms/docs/raw-encryption but I'm guessing you mean the point about the lack of risks. That was something that I tested manually by creating a AES CBC key then revoking all admin permissions other than this one from myself to see if I could take any actions on this key. I could not.

Reply:
Yes, I meant the point about lack of risks. Makes sense!